lib/actions.ts

```diff
@@ -205,7 +213,7 @@ export async function getStandings(): Promise<TeamStanding[]> {
     });
 
     // Calculate statistics based on matches and results
-    matches.forEach((match) => {
+    matches.forEach((match: { nombre_pareja1: string; nombre_pareja2: string; resultado: { set_pareja1: number | null; set_pareja2: number | null } | null }) => {
```

Comment:
```suggestion
    matches.forEach((match: Partido) => {
```
Partido es un type, import de PrismaClient